lessons/Algebra2/exploring-quadratic-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,178 @@
+= Exploring Quadratic Models
+
+@description{Students investigate quadratic relationships in data about Fuel Efficiency, using an inquiry-based model of sense-making, hypothesizing, modeling and testing. They are introduced to table transformations and inverse functions, which are used to fit quadratic models onto nonlinear data.}
+
+@lesson-prereqs{exploring-linear-models, lookups}
+
+@keywords{quadratic, square, coefficient}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Read and interpret real-world data, presented in a scatter plot
+* Model quadratic relationships using functions
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Pyret to model quadratic relationships in data
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* All students should pull up their @online-exercise{program-list} and, open the @starter-file{alg2-fuel-efficiency}.
+
+| Key Points For The Facilitator
+|
+* Unlike linear functions, which are governed by slope and y-intercept, quadratic functions are governed by the bevahior of _three_ factors.
+* Students should gain an intuition for these factors by playing with them first in Desmos, then use them to fit a model in Pyret.
+|===
+
+== Looking for Patterns @duration{20 minutes}
+
+=== Overview
+Students explore the @starter-file{alg2-fuel-efficiency}, and create scatter plots to search for relationships between columns. They share and discuss their findings with the class, discovering the limitations of linear models.
+
+=== Launch
+
+When you're driving, what speed will save you the most money on gas?
+
+[.lesson-instruction]
+- Open the @starter-file{alg2-fuel-efficiency}
+- From the File menu, select "Save a Copy", and click "Run".
+- Working in pairs or small groups, complete questions 1-9 of @printable-exercise{pages/exploring-mpg.adoc}.
```

Comment:
the cut off on the workbook page happens after question 10. not sure whether you meant for this to say 1-9 or were hoping for a second stop after question 10

Reply:
fixed